assignments/favourite_movie/favourite_movie.js

```diff
@@ -2,6 +2,12 @@ const movies = [];
 
 function favouriteMovie(operation, movie) {
     // Write your code here
+    if (operation === "add") {
+        movies.push(movie)
+    }
+    else if (operation === "remove") {
```

Comment:
You always removing last movie. What if there the movie doesn't present at last and in between?